.travis.yml

```diff
@@ -3,7 +3,7 @@ dist: xenial
 
 language: go
 go:
-- 1.15.x
+- 1.15.2
```

Comment:
why make it fixed to 1.15.2?

Reply:
Reproducible builds, see https://github.com/operator-framework/operator-sdk/pull/4034#discussion_r508854179. I am reconsidering this decision though.